README.md

```diff
@@ -24,88 +24,69 @@ We also host native packages for Linux on [Package Cloud](https://packagecloud.i
+
 To start a single node, you can then run:
 ```
-bin/Release/EventStore.ClusterNode/net471/EventStore.ClusterNode.exe --db ../db --log ../logs
+dotnet bin/Release/EventStore.ClusterNode/netcoreapp3.1/<runtime identifier>/EventStore.ClusterNode.dll --db /path/to/db --log /path/to/logs
 ```
 
-You'll need to launch the node with `mono` on Linux or Mac OS X.
-
-_Note: The build system has changed after version `4.1.1-hotfix1`, therefore the above instructions will not work for old releases._
+_Note: The build system has changed after version `5.0.5`, therefore the above instructions will not work for older releases._
 
 ### Running the tests
 You can launch the tests as follows:
 
-#### EventStore Core tests
 ```
-dotnet test src/EventStore.Core.Tests/EventStore.Core.Tests.csproj -- RunConfiguration.TargetPlatform=x64
+dotnet test src/EventStore.sln
 ```
 
-#### EventStore Projections tests
+## Building the EventStore Client / Grpc Client / Embedded Client
+You can build the different clients by following the steps below. This will generate a nuget package file (.nupkg) that you can include in your project.
+#### Client
 ```
-dotnet test src/EventStore.Projections.Core.Tests/EventStore.Projections.Core.Tests.csproj -- RunConfiguration.TargetPlatform=x64
+dotnet pack -c Release src/EventStore.ClientAPI/EventStore.ClientAPI.csproj /p:Version=6.0.0
 ```
 
-## Building the EventStore Client / Embedded Client
-You can build the client / embedded client with the steps below. This will generate a nuget package file (.nupkg) that you can include in your project.
-#### Client
+#### Grpc Client
 ```
-dotnet pack -c Release src/EventStore.ClientAPI/EventStore.ClientAPI.csproj /p:Version=5.0.0
+dotnet pack -c Release src/EventStore.Grpc/EventStore.Grpc.csproj /p:Version=6.0.0
 ```
 
 #### Embedded Client
 ```
-dotnet pack -c Release src/EventStore.ClientAPI.Embedded/EventStore.ClientAPI.Embedded.csproj /p:Version=5.0.0
+dotnet pack -c Release src/EventStore.ClientAPI.Embedded/EventStore.ClientAPI.Embedded.csproj /p:Version=6.0.0
 ```
 
 
 ## Building the EventStore web UI
-The web UI is prebuilt and the files are located under [src/EventStore.ClusterNode.Web/clusternode-web](src/EventStore.ClusterNode.Web/clusternode-web).
-If you want to build the web UI, please consult this [repository](https://github.com/EventStore/EventStore.UI) which is also a git submodule of the current repository located under `src/EventStore.UI`.
+The [web UI repository](https://github.com/EventStore/EventStore.UI) is a git submodule of the current repository located under `src/EventStore.UI`.
+
+The web UI is prebuilt and the files are located in [src/EventStore.ClusterNode.Web/clusternode-web](src/EventStore.ClusterNode.Web/clusternode-web). However, if you still want to build the latest web UI, there is a parameter in the `build.sh` and `build.ps1` scripts to allow you to do so.
```

Comment:
We could provide an example of what the parameter is so that people don't have to go dig around in what this parameter is?
i.e.
```
if you still want to build the latest web UI, you can do so by specifying `-BuildUI yes`.
```

Reply:
@pgermishuys thanks for the suggestion! i've updated the sentence to include the argument name